### PR TITLE
recover secret

### PR DESCRIPTION
**Pull request by ameba23**

### CODE REVIEW

**Comment by ameba23** [comment]
some context of what im trying to do:

kieran has kindly sent me a shard back, in amoungst several other reply messages with the same rootid.  some of these are pure jibberish (actually quite interesting jibberish) and wont pass as valid shards, others are minor modifiactions of the original shard, meaning that without the other shards i have no way to tell which one is the one i sent him, and will need to keep hold of all of them even though only one is correct.   

Currently this breaks the progress bar (as it appears there are more shards than shardholders) and also makes it difficult for me to see if quorum is met.  

So i need to record the number of shards from unique shardholders, and use that in the progressbar and  to compare with quorum before attemting to recombine.

**Comment by ameba23** [comment]
aha so it seems this work has already been done by @mixmix :
`const recordsWithReplies = records.filter(r => r.replies.length > 1)`
gives only records with at least one reply.

but strangely, since this 'minor refactor' commit the progress bar is not displaying at all.

**Comment by ameba23** [comment]
so im still having issues with the progress bar.
for some reason the `records` array being used to determine the value is not an array of objects but a single shardRecord object itself.  Im not sure why.

Other than that slowly getting there.  I guess i should really make this a recombine view rather than begin part of the ritual show view. 

There's other problems but gonna call it a day for today.

File: views/recombine/show.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+          when(recombineError,
+            h('div.recombineError', [
+              h('p', 'Error recombining!'),
+              h('button -subtle', { 'ev-click': () => secretOpen.set(false) }, 'OK')
+            ]),
+
+            h('div.secret', [
+              h('h3', 'Secret recovered successfully:'),
+              // h('span', secret),
+              h('button -subtle', { 'ev-click': () => copyToClipboard('') }, 'Copy to clipboard'),
```
Comment (KGibb8) [inline views/recombine/show.js]: Font awesome icons are available :smile: 

`<i class="fas fa-copy"></i>` gives a sick lil' copy icon.

**Comment by ameba23** [comment]
since bumbing scuttle-dark-crystal things have got a bit messy with other parts of this module calling things that dont exist anymore.  i tried to rebase master and got into a bit of a mess of conflicts, so i have started a new branch and re-commited the relevant bits.

**Pull request closed without merging**